src/main/java/frc/robot/RobotContainer.java

```diff
@@ -48,7 +48,7 @@ public class RobotContainer {
 
 
   /* Drive Controls */
-
+  private AutoAlignMethods autoAlignMethods;
```

Comment:
We probably don't need an instance of this, since we're calling the methods statically.

Reply:
removed